contracts/lending/ttoken/TToken_V1.sol

```diff
@@ -251,7 +243,11 @@ contract TToken_V1 is ITToken {
      * @notice Redeem supplied Teller token underlying value.
      * @param amount The amount of Teller tokens to redeem.
      */
-    function redeem(uint256 amount) external override {
+    function redeem(uint256 amount)
+        external
+        override
+        nonReentry(LendingLib.ID)
```

Comment:
This ID is meant for the `LendingFacet` and should not be using the library for such on the `TToken` which uses different storage.